src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -1966,6 +2002,32 @@ void HMICapabilitiesImpl::set_ccpu_version(const std::string& ccpu_version) {
   ccpu_version_ = ccpu_version;
 }
 
+bool HMICapabilitiesImpl::matches_ccpu_version(
+    const std::string& ccpu_version) {
+  return ccpu_version_ == ccpu_version;
+}
+
+void HMICapabilitiesImpl::update_capabilities_depending_on_version(
+    const std::string& ccpu_version) {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  if (matches_ccpu_version(ccpu_version)) {
+    app_mngr_.SetHMICooperating(true);
+    app_mngr_.OnSendGetCapabilitiesForInterface();
+    return;
+  }
+
+  set_ccpu_version(ccpu_version);
+  DeleteCachedCapabilitiesFile();
+
+  if (false == LoadCapabilitiesFromFile()) {
```

Comment:
if (!LoadCapabilitiesFromFile ...

Reply:
@LuxoftAKutsan done in [481aa86](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/481aa864a58d4bcf8ac488cac83e114f0dc29c40)